cpp/scenario/gazebo/src/GazeboSimulator.cpp

```diff
@@ -171,6 +173,18 @@ bool GazeboSimulator::initialize()
         return false;
     }
 
+    auto cb = [&](int sig) {
+        this->close();
+        exit(sig);
+    };
+
+    // Setup signals callbacks.
+    // It must be done after the creation of the simulator since
+    // we oveeride override their callbacks.
```

Comment:
```suggestion
    // we override their callbacks.
```

Reply:
Thanks! Fixed in https://github.com/robotology/gym-ignition/compare/4ac1c19cfc61c75681c181b79d4790b7712e5de1..8eb570587bec2790c062c3eed651905f5b431d2d